### PR TITLE
add basic test

### PR DESCRIPTION
**Pull request by mdawn**
## [Test External PR to Add cors rules](https://www.pivotaltracker.com/story/show/174803012)

This appropriates [this suggested change](https://github.com/trussworks/terraform-aws-s3-private-bucket/pull/165) and adds a terratest for our troubles.

Changes proposed:
- adds terratest

### CODE REVIEW

**Comment by mdawn** [comment]
For context here, we're passing a list of a map that contains a list (cors structure submitted as `list(any)` in TF) through to that terraform options struct, which is a map of strings to interface...

File: variables.tf [hunk 1]
```diff
@@ -27,25 +27,25 @@ variable "tags" {
   type        = map(string)
 }
 
-variable "enable_bucket_inventory" {
+variable enable_bucket_inventory {
```
Comment (dynamike) [inline variables.tf:30]: There's a mix of quoted and unquoted variables in variables.tf. Can we just continue using quoted?

Comment (ralren) [reply variables.tf:30]: I will change.

Comment (mdawn) [reply variables.tf]: It's done, but why do you prefer quoted?

Comment (dynamike) [reply variables.tf]: I don't really have a preference, just want to maintain consistency

**Review by dynamike — APPROVED** [review]

**Pull request merged**